scripts/elasticsearch-operations.js

```diff
@@ -110,6 +110,43 @@ const deleteIndex = async () => {
     }
 };
 
+const addAlias = async () => {
```

Comment:
I'd suggest adding a comment/docs describing what this function does.

I know that the rest of this file has almost no documentation, but I feel like this one deserves it since It will be mentioned as the path to upgrade for breaking change on the release notes.

All the other functions are extras that I'm not sure if they are ever used (I didn't even know this file existed).

Reply:
Good point! I'll add documentation for it in subsequent PR. I was not aware of this file exited before either, I'll bring it up in tmr's standup to see if we still want to keep them.